### PR TITLE
WT-6245 Instantiate both update and tombstone for prepared tombstone

### PR DESCRIPTION
**Pull request by kommiharibabu**
When the prepared update is a tomnbstone that is written to the datastore,
while instantiating the data store, make sure that instantiate both
update and tombstone to avoid replacing the on-disk value.

### CODE REVIEW

File: src/btree/bt_page.c [hunk 1]
```diff
@@ -607,29 +609,46 @@ __inmem_row_leaf(WT_SESSION_IMPL *session, WT_PAGE *page)
...
+                /* Take the value from the original page cell. */
+                WT_RET(__wt_page_cell_data_ref(session, page, &unpack, &buf));
+
+                WT_RET(__wt_upd_alloc(session, &buf, WT_UPDATE_STANDARD, &upd, &size));
+                upd->durable_ts = unpack.tw.durable_start_ts;
+                upd->start_ts = unpack.tw.start_ts;
+                upd->txnid = unpack.tw.start_txn;
+
+                /*
+                 * If the prepared update is of tombstone, instantiating both update and tombstone
```
Comment (luke-pearson) [inline src/btree/bt_page.c:621]: Wording: If the prepared update is a tombstone.

Comment (kommiharibabu) [reply src/btree/bt_page.c]: The comment is updated. Check it again.

Comment (luke-pearson) [reply src/btree/bt_page.c]: You still say is of tombstone, my point was that it should read, "is a tombstone"

Comment (kommiharibabu) [reply src/btree/bt_page.c]: Fixed. Thanks.

**Pull request merged**